packages/rest/src/manager.ts

```diff
@@ -175,7 +175,7 @@ export function createRestManager(options: CreateRestManagerOptions): RestManage
           form.append(`file${i}`, options.files[i].blob, options.files[i].name)
         }
 
-        form.append('payload_json', JSON.stringify(snakelize({ ...options.body, files: undefined })))
+        form.append('payload_json', JSON.stringify(rest.changeToDiscordFormat({ ...options.body, files: undefined })))
```

Comment:
Would like to have a comment above this line, explaining why we use rest.changeToDiscordFormat and not snakelize to avoid someone accidentally changing to snakelize here

Reply:
I have added a comment for it, let me know if it gives an explanation that is understandable